hvac/api/system_backend/wrapping.py

```diff
@@ -27,3 +27,24 @@ def unwrap(self, token=None):
             url=api_path,
             json=params,
         )
+
+    def wrap(self, payload=dict(), ttl=60):
+        """Wraps a serializable dictionary inside a wrapping token.
+
+        Supported methods:
+            POST: /sys/wrapping/wrap. Produces: 200 application/json
+
+        :param payload: Specifies the data that should be wrapped inside the token.
+        :type payload: dict
+        :param ttl: The TTL of the returned wrapping token.
+        :type ttl: int
+        :return: The JSON response of the request.
+        :rtype: dict
+        """
+
```

Comment:
This presumably runs into the 'ole [mutable default argument "gotcha"](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments). To accomplish what you're after (i.e., an empty `payload` dict when no argument explicitly provided), it would be better to go with something like this:

```suggestion
    def wrap(self, payload=None, ttl=60):
        """Wraps a serializable dictionary inside a wrapping token.

        Supported methods:
            POST: /sys/wrapping/wrap. Produces: 200 application/json

        :param payload: Specifies the data that should be wrapped inside the token.
        :type payload: dict
        :param ttl: The TTL of the returned wrapping token.
        :type ttl: int
        :return: The JSON response of the request.
        :rtype: dict
        """
        if payload is None:
                payload = dict()
```

Reply:
Agreed, though I would recommend `payload = {}`